catalog/plugin/catalog-plugin-querymonitor/src/main/java/ddf/catalog/querymonitor/QueryMonitorPluginImpl.java

```diff
@@ -0,0 +1,139 @@
+/**
+ * Copyright (c) Codice Foundation
+ * <p>
+ * This is free software: you can redistribute it and/or modify it under the terms of the GNU Lesser
+ * General Public License as published by the Free Software Foundation, either version 3 of the
+ * License, or any later version.
+ * <p>
+ * This program is distributed in the hope that it will be useful, but WITHOUT ANY WARRANTY; without
+ * even the implied warranty of MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the GNU
+ * Lesser General Public License for more details. A copy of the GNU Lesser General Public License
+ * is distributed along with this program and can be found at
+ * <http://www.gnu.org/licenses/lgpl.html>.
+ */
+package ddf.catalog.querymonitor;
+
+import java.util.Map;
+import java.util.UUID;
+import java.util.concurrent.ConcurrentHashMap;
+
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+import ddf.catalog.operation.QueryRequest;
+import ddf.catalog.operation.QueryResponse;
+import ddf.catalog.plugin.PluginExecutionException;
+import ddf.catalog.plugin.StopProcessingException;
+import ddf.catalog.source.Source;
+
+public class QueryMonitorPluginImpl implements QueryMonitorPlugin {
+
+    boolean removeSearchAfterComplete = true;
+
+    private ConcurrentHashMap<UUID, ActiveSearch> activeSearches =
+            new ConcurrentHashMap<UUID, ActiveSearch>();
```

Comment:
You can remove the type parameters in `new ConcurrentHashMap<UUID, ActiveSearch>()`.